packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -104,20 +113,19 @@ function validate(event: any) {
       <div class="space-y-2">
         <label
           for="httpProxy"
-          class="pt-4 mb-2 font-medium {proxyState ? 'text-[var(--pd-invert-content-card-text)]' : 'text-gray-900'}"
-          >Bypass proxy settings for these hosts and domains:</label>
+          class="pt-4 mb-2 font-medium {proxyState === 1
+            ? 'text-[var(--pd-invert-content-card-text)]'
+            : 'text-gray-900'}">Bypass proxy settings for these hosts and domains:</label>
         <Input
           name="noProxy"
           id="noProxy"
-          disabled={!proxyState}
+          disabled={proxyState !== 1}
```

Comment:
```suggestion
          disabled={proxyState !== ProxyState.PROXY_MANUAL}
```

Copy paste from line 86